src/wp-includes/script-loader.php

```diff
@@ -121,7 +125,7 @@ function wp_default_packages_vendor( $scripts ) {
 		$scripts->add( $handle, $path, $dependencies, $version, 1 );
 	}
 
-	$scripts->add( 'wp-polyfill', null, array( 'wp-polyfill' ) );
+	$scripts->add( 'wp-polyfill', null, array( 'regenerator-runtime', 'wp-polyfill-built' ) );
```

Comment:
Is there a strong reason to avoid merging these two scripts in to one and retain the single script alias? 

The though behind this question is two fold:
* next time WP changes polyfill libraries, avoiding a `-built-final-final-FINAL-2` type of situation
* performance cost/benefit of sending one script vs two

Reply:
I don't mind having two scripts here. In the future, we might be able to drop `regenerator-runtime` for core scripts and make it opt-in for 3rd projects for backward compatibility. I'm not sure if that is going to be a simple task though.